rtgui/preferences.cc

```diff
@@ -947,7 +947,7 @@ Gtk::Widget* Preferences::getGeneralPanel()
     complexitylocal->append(M("PREFERENCES_COMPLEXITY_EXP"));
     complexitylocal->append(M("PREFERENCES_COMPLEXITY_NORM"));
     complexitylocal->append(M("PREFERENCES_COMPLEXITY_SIMP"));
-    complexitylocal->set_active(1);
+    complexitylocal->set_active(2);
```

Comment:
@Desmis 
In `option.cc` file, default value for `complexity` shall also be updated to 2 to change the default mode ;)
Pierre